README.md

```diff
@@ -59,7 +59,7 @@ The directory where your Brewfile is located.
 
 ## Dependencies
 
-None.
+* [elliotweiser.osx-command-line-tools](https://galaxy.ansible.com/elliotweiser/osx-command-line-tools/)
```

Comment:
Ah... to actually install the CLI tools. In this case, is there a missing commit that uses the role?